docs/pages/sdk-reference/core/type-aliases/RunAsyncOptions.mdx

```diff
@@ -71,6 +72,34 @@ If omitted, the function won't be retried.
 
 ***
 
+### shouldRetry()?
+
+```ts
+optional shouldRetry: (error) => boolean;
+```
+
+Defined in: [packages/sdk/src/core/utils.ts:154](https://github.com/ecp-eth/comments-monorepo/blob/main/packages/sdk/src/core/utils.ts#L154)
+
+A function to determine if the function should be retried.
+
+By default, the function will always retry until the number of retries is reached.
```

Comment:
what's the number of retries?

Reply:
it seems that perhaps this function should be named differently, as it can return true but with retries 0 it doesn't retry. Perhaps `retryCondition` like Axios retry uses?

Reply:
done